subiquitycore/models/network.py

```diff
@@ -45,6 +46,11 @@ class Networkdev:
     def __init__(self, net_info, configuration):
         self._net_info = net_info
         self._configuration = configuration
+        if self.type == 'vlan':
+            # probably should parse /proc/self/net/vlan/config instead
```

Comment:
Weeellll reallllly probert's _rtnetlink stuff should be looking out for this?

Reply:
Quite.... but it doesn't do that now =/

Reply:
https://trello.com/c/ayU1R0Al